.cookiecutter/includes/setuptools/install_requires

```diff
@@ -1 +1,2 @@
 pip-tools
+importlib_metadata;python_version<"3.8."
```

Comment:
Conditionally depend on [importlib-metadata](https://pypi.org/project/importlib-metadata/) in Python 3.7 only.